index.js

```diff
@@ -24,7 +24,7 @@ var PACKAGE_NAME = require('./package.json').name;
  * A webpack loader that resolves absolute url() paths relative to their original source file.
  * Requires source-maps to do any meaningful work.
  * @param {string} content Css content
- * @param {object} sourceMap The source-map
+ * @param {object|String} sourceMap The source-map
```

Comment:
**nit** Not sure what is more correct `string` or `String`.

I don't recall why the following line has both (?), it seems wrong to me. Feel free to fix that too if you agree.